packages/airnode-validator/src/receipt/receipt.ts

```diff
@@ -11,8 +12,15 @@ export const airnodeWalletSchema = z.object({
 export type AirnodeWallet = SchemaType<typeof airnodeWalletSchema>;
 
 export const deploymentSchema = z.object({
-  // TODO: This must match the version of validator
-  nodeVersion: z.string(),
+  nodeVersion: z.string().superRefine((version, ctx) => {
```

Comment:
This is the same refinement check as in the `config.ts` validation, but I didn't want to abstract this to a "common" file just yet...